### PR TITLE
Linters setup and Code Review

### PR DESCRIPTION
**Pull request by richy4dev**
In this branch, I setup the linters to check for linters error.
I also made corrections to all incorrect codes and bugs after reviewing.

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+      <div class="form-group">
+        <label id="name-label" for="name">Fullname</label>
+        <input
+          type="text"
+          name="name"
+          id="name"
+          class="form-control"
+          placeholder="Enter your name"
+          required
+        />
```
Comment (richy4dev) [inline Index.html:29]: Kindly close the "input" element.

File: Index.html [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    </div>
+          
+      <div class="form-group">
+        <p>
+          What course are you interested in?
+          <span class="clue">(Check all that apply)</span>
+        </p>
+  
+        <label
+          ><input
```
Comment (richy4dev) [inline Index.html:78]: Kindly move the bracket that closes the label input upwards to line 77

File: Index.html [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+        <label>
+          <input
+            name="prefer"
+            value="Back-end Engineering"
+            type="checkbox"
+            class="input-checkbox"
+          />Back-end Engineering</label
+        >
+        <label
+          ><input
```
Comment (richy4dev) [inline Index.html:94]: Just like for Line 77, do the same here

Comment (richy4dev) [reply Index.html:94]: there are repetitions of this on lines 100, 102, 110, 118 and 124 of your HTML code.

This makes for a very clean and organized code.

**Review by ishimwezachee — APPROVED** [review]
Hi @richy4dev , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 Project was reviewed 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**